src/pages/IndividualArticlePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { getIndividualArticle } from '../utilsAPI';
+import { useEffect, useState } from 'react';
+
+const IndividualArticlePage = () => {
+  const { article_id } = useParams();
```

Comment:
Awesome use of use Params to ensure this page works if a user goes directly to the article though the link rarther than just passing props down from the articles list 💯